src/afl-fuzz.c

```diff
@@ -1352,6 +1352,17 @@ int main(int argc, char **argv_orig, char **envp) {
   if (getenv("AFL_DISABLE_FAVS"))     afl->disable_afl_default_favorites       = 1;
   if (getenv("AFL_DISABLE_RP"))       afl->disable_randomized_fuzzing_params   = 1;
 
+  // initialize with default values if we don't want to radomize ruzzing params
+  if (afl->disable_randomized_fuzzing_params) {
```

Comment:
Could you just call `reset_fuzzing_params(afl)` here independently of whether `afl->disable_randomized_fuzzing_params` is set?

Reply:
Fixed

Reply:
Great! 👍 Thanks!